### PR TITLE
[Node] (Crypto) {KeyObject} Fix typo symmetricSize -> symmetricKeySize

### PR DESCRIPTION
**Pull request by CarlosKlapp**
- [X ] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

### CODE REVIEW

**Comment by Flarna** [comment]
Could you please add a test?

**Comment by SimonSchick** [comment]
Pls also fix in v12.

**Review by SimonSchick — APPROVED** [review]

**Comment by CarlosKlapp** [comment]
I fixed the typo in V12 and I added a test.

Are the test cases actually executed? For example, 
`ts-node types/node/test/assert.ts`
fails along with other files I tried.

Without executing the JS code, it seems impossible to verify the shape of the object returned by the JS function actually matches the TS definition. Or am I missing something?

Thank you in advance.

**Comment by Flarna** [comment]
There is no JS code executed in the tests. The test files are transpiled and [dtslinted](https://github.com/microsoft/dtslint).
Therefore checking is done by e.g. assigning the return value of some function to a variable which has a specific type which wont transpile if there is a mismatch.
dtslint offers the `$ExpectType`,  `$ExpectError` assertions on top.

Taking a look into your tests added:
```
const keyObject = crypto.createSecretKey(Buffer.from('asdf')); // $ExpectType KeyObject
keyObject instanceof crypto.KeyObject;
assert.equal(keyObject.symmetricKeySize, 4);
assert.equal(keyObject.type, "secret");
```
The first line verifies that `crypto.createSecretKey` is a function accepting a `Buffer` as argument and returns a `KeyObject`.
The second line verifies that `crypto.KeyObject` has a constructor.
The third line verifies that `keyObject` has a property named `symmetricKeySize` but the type or the value is not verified.

**Review by Flarna — APPROVED** [review]

**Review by galkin — APPROVED** [review]

File: types/node/test/crypto.ts [hunk 1]
```diff
@@ -561,7 +561,10 @@ import { promisify } from 'util';
 }
 
 {
-    crypto.createSecretKey(Buffer.from('asdf'));
+    const keyObject = crypto.createSecretKey(Buffer.from('asdf')); // $ExpectType KeyObject
+    keyObject instanceof crypto.KeyObject;
+    assert.equal(keyObject.symmetricKeySize, 4);
+    assert.equal(keyObject.type, "secret");
```
Comment (ExE-Boss) [inline types/node/test/crypto.ts]: Maybe also copy this to `types/node/v12/test/crypto.ts`.

**Review by ExE-Boss — APPROVED** [review]
This is correct: <https://nodejs.org/api/crypto.html#crypto_keyobject_symmetrickeysize>

**Pull request merged**